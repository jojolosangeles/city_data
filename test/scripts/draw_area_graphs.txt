all_areas = expected/all_areas.csv
all_areas.filter Area Name
all_areas.Area Name.bar X|Year:O|Year, Y|count()|Number of Accidents
all_areas.filter Victim Sex
all_areas.Area Name.bar X|Year:O|Year, Y|count()|Number of Accidents

#
#  To see accident count by Area Name, we need to get rid of the unnecessary data
#
all_areas.drop Unnamed: 0, Date Occurred, Time Occurred, Victim Age, Address, Victim Descent
all_areas.drop Victim Sex, Cross Street, Date, Weekday, Hour

all_areas.bar X|Area Name:N|Area Name, Y|count()|Number of Accidents