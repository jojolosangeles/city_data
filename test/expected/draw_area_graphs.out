Test: scripts/draw_area_graphs.txt
> all_areas = expected/all_areas.csv
Loaded expected/all_areas.csv into DataFrame 'all_areas'
> all_areas.filter Area Name
Created DataFrame all_areas.Area_Name.Central, saved CSV
Created DataFrame all_areas.Area_Name.77th_Street, saved CSV
Created DataFrame all_areas.Area_Name.Pacific, saved CSV
Created DataFrame all_areas.Area_Name.Topanga, saved CSV
Created DataFrame all_areas.Area_Name.Southwest, saved CSV
Created DataFrame all_areas.Area_Name.Foothill, saved CSV
Created DataFrame all_areas.Area_Name.Newton, saved CSV
Created DataFrame all_areas.Area_Name.Hollywood, saved CSV
Created DataFrame all_areas.Area_Name.N_Hollywood, saved CSV
Created DataFrame all_areas.Area_Name.Northeast, saved CSV
Created DataFrame all_areas.Area_Name.Hollenbeck, saved CSV
Created DataFrame all_areas.Area_Name.West_LA, saved CSV
Created DataFrame all_areas.Area_Name.Southeast, saved CSV
Created DataFrame all_areas.Area_Name.Devonshire, saved CSV
Created DataFrame all_areas.Area_Name.Mission, saved CSV
Created DataFrame all_areas.Area_Name.Harbor, saved CSV
Created DataFrame all_areas.Area_Name.Wilshire, saved CSV
Created DataFrame all_areas.Area_Name.Rampart, saved CSV
Created DataFrame all_areas.Area_Name.Olympic, saved CSV
Created DataFrame all_areas.Area_Name.West_Valley, saved CSV
Created DataFrame all_areas.Area_Name.Van_Nuys, saved CSV
> all_areas.Area Name.bar X|Year:O|Year, Y|count()|Number of Accidents
Making bar chart for DataFrame all_areas.Area_Name.Central
Saved Bar Graph PNG './actual/all_areas.Area_Name.Central.png'
Making bar chart for DataFrame all_areas.Area_Name.77th_Street
Saved Bar Graph PNG './actual/all_areas.Area_Name.77th_Street.png'
Making bar chart for DataFrame all_areas.Area_Name.Pacific
Saved Bar Graph PNG './actual/all_areas.Area_Name.Pacific.png'
Making bar chart for DataFrame all_areas.Area_Name.Topanga
Saved Bar Graph PNG './actual/all_areas.Area_Name.Topanga.png'
Making bar chart for DataFrame all_areas.Area_Name.Southwest
Saved Bar Graph PNG './actual/all_areas.Area_Name.Southwest.png'
Making bar chart for DataFrame all_areas.Area_Name.Foothill
Saved Bar Graph PNG './actual/all_areas.Area_Name.Foothill.png'
Making bar chart for DataFrame all_areas.Area_Name.Newton
Saved Bar Graph PNG './actual/all_areas.Area_Name.Newton.png'
Making bar chart for DataFrame all_areas.Area_Name.Hollywood
Saved Bar Graph PNG './actual/all_areas.Area_Name.Hollywood.png'
Making bar chart for DataFrame all_areas.Area_Name.N_Hollywood
Saved Bar Graph PNG './actual/all_areas.Area_Name.N_Hollywood.png'
Making bar chart for DataFrame all_areas.Area_Name.Northeast
Saved Bar Graph PNG './actual/all_areas.Area_Name.Northeast.png'
Making bar chart for DataFrame all_areas.Area_Name.Hollenbeck
Saved Bar Graph PNG './actual/all_areas.Area_Name.Hollenbeck.png'
Making bar chart for DataFrame all_areas.Area_Name.West_LA
Saved Bar Graph PNG './actual/all_areas.Area_Name.West_LA.png'
Making bar chart for DataFrame all_areas.Area_Name.Southeast
Saved Bar Graph PNG './actual/all_areas.Area_Name.Southeast.png'
Making bar chart for DataFrame all_areas.Area_Name.Devonshire
Saved Bar Graph PNG './actual/all_areas.Area_Name.Devonshire.png'
Making bar chart for DataFrame all_areas.Area_Name.Mission
Saved Bar Graph PNG './actual/all_areas.Area_Name.Mission.png'
Making bar chart for DataFrame all_areas.Area_Name.Harbor
Saved Bar Graph PNG './actual/all_areas.Area_Name.Harbor.png'
Making bar chart for DataFrame all_areas.Area_Name.Wilshire
Saved Bar Graph PNG './actual/all_areas.Area_Name.Wilshire.png'
Making bar chart for DataFrame all_areas.Area_Name.Rampart
Saved Bar Graph PNG './actual/all_areas.Area_Name.Rampart.png'
Making bar chart for DataFrame all_areas.Area_Name.Olympic
Saved Bar Graph PNG './actual/all_areas.Area_Name.Olympic.png'
Making bar chart for DataFrame all_areas.Area_Name.West_Valley
Saved Bar Graph PNG './actual/all_areas.Area_Name.West_Valley.png'
Making bar chart for DataFrame all_areas.Area_Name.Van_Nuys
Saved Bar Graph PNG './actual/all_areas.Area_Name.Van_Nuys.png'
> all_areas.filter Victim Sex
Created DataFrame all_areas.Victim_Sex.F, saved CSV
Created DataFrame all_areas.Victim_Sex.M, saved CSV
Skipping non-string
Created DataFrame all_areas.Victim_Sex.H, saved CSV
Created DataFrame all_areas.Victim_Sex.X, saved CSV
Created DataFrame all_areas.Victim_Sex.N, saved CSV
> all_areas.Area Name.bar X|Year:O|Year, Y|count()|Number of Accidents
Making bar chart for DataFrame all_areas.Area_Name.Central
Saved Bar Graph PNG './actual/all_areas.Area_Name.Central.png'
Making bar chart for DataFrame all_areas.Area_Name.77th_Street
Saved Bar Graph PNG './actual/all_areas.Area_Name.77th_Street.png'
Making bar chart for DataFrame all_areas.Area_Name.Pacific
Saved Bar Graph PNG './actual/all_areas.Area_Name.Pacific.png'
Making bar chart for DataFrame all_areas.Area_Name.Topanga
Saved Bar Graph PNG './actual/all_areas.Area_Name.Topanga.png'
Making bar chart for DataFrame all_areas.Area_Name.Southwest
Saved Bar Graph PNG './actual/all_areas.Area_Name.Southwest.png'
Making bar chart for DataFrame all_areas.Area_Name.Foothill
Saved Bar Graph PNG './actual/all_areas.Area_Name.Foothill.png'
Making bar chart for DataFrame all_areas.Area_Name.Newton
Saved Bar Graph PNG './actual/all_areas.Area_Name.Newton.png'
Making bar chart for DataFrame all_areas.Area_Name.Hollywood
Saved Bar Graph PNG './actual/all_areas.Area_Name.Hollywood.png'
Making bar chart for DataFrame all_areas.Area_Name.N_Hollywood
Saved Bar Graph PNG './actual/all_areas.Area_Name.N_Hollywood.png'
Making bar chart for DataFrame all_areas.Area_Name.Northeast
Saved Bar Graph PNG './actual/all_areas.Area_Name.Northeast.png'
Making bar chart for DataFrame all_areas.Area_Name.Hollenbeck
Saved Bar Graph PNG './actual/all_areas.Area_Name.Hollenbeck.png'
Making bar chart for DataFrame all_areas.Area_Name.West_LA
Saved Bar Graph PNG './actual/all_areas.Area_Name.West_LA.png'
Making bar chart for DataFrame all_areas.Area_Name.Southeast
Saved Bar Graph PNG './actual/all_areas.Area_Name.Southeast.png'
Making bar chart for DataFrame all_areas.Area_Name.Devonshire
Saved Bar Graph PNG './actual/all_areas.Area_Name.Devonshire.png'
Making bar chart for DataFrame all_areas.Area_Name.Mission
Saved Bar Graph PNG './actual/all_areas.Area_Name.Mission.png'
Making bar chart for DataFrame all_areas.Area_Name.Harbor
Saved Bar Graph PNG './actual/all_areas.Area_Name.Harbor.png'
Making bar chart for DataFrame all_areas.Area_Name.Wilshire
Saved Bar Graph PNG './actual/all_areas.Area_Name.Wilshire.png'
Making bar chart for DataFrame all_areas.Area_Name.Rampart
Saved Bar Graph PNG './actual/all_areas.Area_Name.Rampart.png'
Making bar chart for DataFrame all_areas.Area_Name.Olympic
Saved Bar Graph PNG './actual/all_areas.Area_Name.Olympic.png'
Making bar chart for DataFrame all_areas.Area_Name.West_Valley
Saved Bar Graph PNG './actual/all_areas.Area_Name.West_Valley.png'
Making bar chart for DataFrame all_areas.Area_Name.Van_Nuys
Saved Bar Graph PNG './actual/all_areas.Area_Name.Van_Nuys.png'
> 
> #
> #  To see accident count by Area Name, we need to get rid of the unnecessary data
> #
> all_areas.drop Unnamed: 0, Date Occurred, Time Occurred, Victim Age, Address, Victim Descent
Dropped ['Unnamed: 0', 'Date Occurred', 'Time Occurred', 'Victim Age', 'Address', 'Victim Descent'] from DataFrame 'all_areas'
> all_areas.drop Victim Sex, Cross Street, Date, Weekday, Hour
Dropped ['Victim Sex', 'Cross Street', 'Date', 'Weekday', 'Hour'] from DataFrame 'all_areas'
> 
> all_areas.bar X|Area Name:N|Area Name, Y|count()|Number of Accidents
Making bar chart for DataFrame all_areas
Saved Bar Graph PNG './actual/all_areas.png'
